nano/node/active_transactions.hpp

```diff
@@ -71,13 +72,20 @@ class cementable_account final
 	uint64_t blocks_uncemented{ 0 };
 };
 
-class confirmed_set_info final
+class election_timepoint final
 {
 public:
 	std::chrono::steady_clock::time_point time;
 	nano::qualified_root root;
 };
 
+typedef boost::multi_index_container<
```

Comment:
`typedef` is legacy c++, aliases are preferred, i.e `using ordered_elections_timepoint = ... `

Reply:
Thanks, changed and moved to inside the class as it's not needed elsewhere.